lib/cocoapods/installer/analyzer/target_generator.rb

```diff
@@ -0,0 +1,397 @@
+module Pod
+  class Installer
+    class Analyzer
+      class TargetGenerator
+        # @return [String] The version of iOS which requires binaries with only 64-bit architectures
+        #
+        IOS_64_BIT_ONLY_VERSION = Version.new('11.0')
```

Comment:
`.freeze`?

Reply:
ah, now I remember why I didn't do this - comparing versions with `>=` triggers a lazy initializer on `_segments` inside `Version`, which is a modification. so basically comparing one version with another modifies the version instance :/